src/components/transaction/transaction.component.html

```diff
@@ -1,8 +1,8 @@
 <div class="transaction">
   <div class="details">
-    Transactions Number
+    {{transaction.title}}
   </div>
   <div class="amount">
-    12.13
+    {{transaction.amount}}
   </div>
```

Comment:
Check if the transaction is  positive or negative and add IN (green) or OUT (red) as displayed in the UI prototype.

Also, please put this in a ion-card that the user can scroll though. Basically it would be nice if the wallet header were always visible.